classes/class-p4-post-campaign.php

```diff
@@ -350,8 +350,8 @@ public static function css_vars( array $meta ): array {
 			$footer_theme = $meta['campaign_footer_theme'] ?? null;
 
 			if ( 'white' === $footer_theme ) {
-				$default_footer_links_color = $meta['campaign_nav_color'] ?: '#1A1A1A';
-				$footer_links_color         = $meta['footer_links_color'] ?: $default_footer_links_color;
+				$default_footer_links_color = $meta['campaign_nav_color'] ? $meta['campaign_nav_color'] : '#1A1A1A';
```

Comment:
Weird that phpcs flags this, as `$meta['campaign_nav_color'] ?: '#1A1A1A';` is just a shorthand for `$meta['campaign_nav_color'] ? $meta['campaign_nav_color'] : '#1A1A1A';`. I'm not aware of any issues with that syntax, but there might be?

Reply:
The `wpcs` is complaining about ternary shorthand -
`Using short ternaries is not allowed
 (WordPress.PHP.DisallowShortTernary.Found)`